models/zone/src/zone-draft/generator.ts

```diff
@@ -1,5 +1,5 @@
 import { fake, Generator } from '@commercetools-test-data/core';
-import { TZoneDraft } from '..//types';
```

Comment:
😬

Reply:
Fixing that was the right call, correct?

Reply:
Definitely! The face was just for missing it in the first place